src/main/java/frc/robot/commands/intake.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.Intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class intake extends Command {
+	private int maxMotorSpeed = frc.robot.constants.IntakeConstants.maxMotorSpeed;
+	private Intake intake;
+
+	public intake() {}
+
+	// Called when the command is initially scheduled.
+	@Override
+	public void initialize() {
+		intake.setMotorSpeed(maxMotorSpeed);
+	}
+
+	// Called every time the scheduler runs while the command is scheduled.
+	@Override
+	public void execute() {}
+
+	// Called once the command ends or is interrupted.
+	@Override
+	public void end(boolean interrupted) {
+		intake.stopMotor();
+	}
+
+	// Returns true when the command should end.
+	@Override
+	public boolean isFinished() {
+		return (intake.detectNote());
```

Comment:
no parentheses needed around the return statements, please remove them for visual consistency.